python/doc/examples/graphs/plot_graphs_basics.py

```diff
@@ -216,7 +216,7 @@ def createHSVColor(indexCurve, maximumNumberOfCurves):
 # Create matrices of graphics
 # ---------------------------
 #
-# The library does *not* has objects to create a grid of graphics. However, we can use the `add_subplot` function from Matplotlib.
+# The library provides features to create a grid of graphics. However, we can use the `add_subplot` function from Matplotlib.
```

Comment:
Throughout this examples, I believe "graphic" as a noun should be replaced with "graph". I am not sure "graphic" as a noun exists in English. See https://context.reverso.net/traduction/francais-anglais/graphique

Reply:
Fixed at a27e17e61